app/assets/stylesheets/forever_style_guide/modules/_list.scss

```diff
@@ -48,6 +48,52 @@
   }
 }
 
+.list-group-toggle {
+  .badge {
+    @extend %check-style;
+    padding: 5px 0 0;
+    font-size: font-size('default');
+    color: color('gray-500');
+    @include circle(24px, color('gray-200'));
+
+    &:hover {
+      color: color('primary');
```

Comment:
That's a fair point. I think since this is a grouping of links, visually it's pretty awash in secondary blue as is and I was looking for a bit of differentiation. But I'm totally fine with changing it for better consistency.